src/utils/getAgeDivID.js

```diff
@@ -0,0 +1,38 @@
+const csv = require("csv-parser");
```

Comment:
you can use import statements instead of require, since import statements load asynchronously. also it can save memory since you can use the function you only need (requires uses all functions from a library/file"

`import csv from "csv-parser"`